aws/windows-2019-agent.amd64.json

```diff
@@ -24,7 +24,7 @@
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "hvm",
-          "name": "Windows_Server-2019-English-Core-ContainersLatest-*",
+          "name": "Windows_Server-2004-English-Core-ContainersLatest-*",
```

Comment:
Is 2004 supposed to reference a year? The transition from 2019 to 2004 seems weird to me even though I don't know the difference between hose two

Reply:
> Is 2004 supposed to reference a year? The transition from 2019 to 2004 seems weird to me even though I don't know the difference between hose two

1809 was followed by 1903 then 1909 and then 2004.  the first two digits are the year, last two digits are the month.  Microsoft's way of numbering their semi-annual releases.